stock_picking_move_package_to_package/models/stock_package_level.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPackageLevel(models.Model):
+    _inherit = "stock.package_level"
+
+    package_dest_id = fields.Many2one(
+        "stock.quant.package",
+        "Destination Package",
+        compute="_compute_package_dest_id",
+        inverse="_inverse_package_dest_id",
+        store=True,
+        readonly=False,
+        check_company=True,
+        domain="[('location_id', 'child_of', parent.location_dest_id), "
+        "'|', "
+        "('company_id', '=', False), "
+        "('company_id', '=', company_id)]",
```

Comment:
What if you want to move to an empty package? Then there is no location. You should allow this case also.
```suggestion
        domain="[('company_id', 'in', [False, company_id]), "
        "'|', ('location_id', 'child_of', parent.location_dest_id), "
        "('location_id', '=', False)]",
```